pyproject.toml

```diff
@@ -7,9 +7,7 @@ readme = "README.md"
 packages = [{include = "src/api/*.py"}]
 
 [tool.poetry.dependencies]
-python = "^3.11"
-numpy = "^1.26.4"
-Flask = "^3.0.3"
```

Comment:
@Soguy-fr is this intentional to keep python version at 3.10 instead of 3.11? Also removing numpy and Flask? Can we at least keep numpy and Flask in for now?

Reply:
Hi,
Nothing intentional, I just make some automatic cleaning.

Reply:
hi! perfect then, definitely please merge it in when you get a chance!